CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(CMAKE_CXX_STANDARD 14)
 file(STRINGS version.txt TORCHVISION_VERSION)
 
 option(WITH_CUDA "Enable CUDA support" OFF)
+option(USE_PYTHON "Link to Python when building" OFF)
```

Comment:
Might be worth adding a reference of that on the README similar to what we do for `WITH_CUDA` to explain when and under which conditions this is necessary to be turned on.

Reply:
Thanks! Just added a paragraph in https://github.com/pytorch/vision/pull/5190/commits/a2594f58c4e9d39b71d3d9eba2be31a06053ccab